src/pages/packer/plugins/index.tsx

```diff
@@ -0,0 +1,64 @@
+import DocsView from 'views/docs-view'
+// Imports below are used server-side
+import { getRootDocsPathGenerationFunctions } from 'views/docs-view/utils/get-root-docs-path-generation-functions'
+import { appendRemotePluginsNavData } from 'components/_proxied-dot-io/packer/remote-plugin-docs/server'
+import prepareNavDataForClient from 'layouts/sidebar-sidecar/utils/prepare-nav-data-for-client'
+
+/**
+ * Path relative to the `website` directory of the Packer GitHub repo.
+ *
+ * Note that these are not currently used, as we don't yet support local
+ * preview for the dev dot UI. They've been retained to avoid too
+ * broad of a refactor to utilities shared with dot-io (where local
+ * preview is actively supported).
+ */
+const remotePluginsFile = 'data/plugins-manifest.json'
+
+/**
+ * Since this /plugins landing page does use content from our API,
+ * we can use the same getStaticProps function as all other Dev Dot docs routes,
+ * with some modifications for the sidebar data.
+ */
+const { getStaticProps: baseGetStaticProps } =
+	getRootDocsPathGenerationFunctions('packer', 'plugins')
+
+/**
+ * We tack on some extra plugin data to the result of the base getStaticProps,
+ * in order to get the sidebar on this page showing links to all remote plugins.
+ */
+async function getStaticProps(ctx) {
```

Comment:
For the plugins landing page, we are still fetching MDX content for the page from our API, so in this case we're able to use the same `getStaticProps` base function as elsewhere. We do need to do a bit of extra work to merge in `navData` for plugins. 

This isn't as complex as the finagling in the adjacent `[...page]` file, but I've still tried to be thorough with comments here.